<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgNNCreator</class>
 <widget class="QDialog" name="DlgNNCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="createNN">
      <attribute name="title">
       <string>Создание модели</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Название новой модели</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nnNameLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1,1">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Количество входных параметров</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="inputParamsSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1,1">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Количество скрытых слоёв</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="hiddenLayersSpinBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="hiddenLayersTableView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,1">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Количество выходных параметров</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="OutputParamsSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="nnGraphicsView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="learnNN">
      <attribute name="title">
       <string>Обучение модели</string>
      </attribute>
      <widget class="QPushButton" name="chooseFileBtn">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>50</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Выбрать файл</string>
       </property>
      </widget>
      <widget class="QLabel" name="chooseFIleLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Файл</string>
       </property>
      </widget>
      <widget class="QPushButton" name="viewDataBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Просмотр данных</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
